lib/spack/spack/reporters/cdash.py

```diff
@@ -62,7 +62,6 @@ class CDash(Reporter):
 
     def __init__(self, args):
         Reporter.__init__(self, args)
-        tty.set_verbose(args.verbose)
```

Comment:
I'm assuming this is the primary issue behind the failure in https://github.com/spack/spack/pull/27335 (https://github.com/spack/spack/runs/4181356619?check_suite_focus=true) (combined with your pointer to https://github.com/python/cpython/commit/a18d52269ab6071a605d6c72f6af585a4c533ca4): i.e. the verbosity from `main` started overriding the verbosity setting on `spack install`.